ci/testkomodo.sh

```diff
@@ -0,0 +1,14 @@
+
+copy_test_files () {
+    cp -r $CI_SOURCE_ROOT/tests $CI_TEST_ROOT/tests
+    cp $CI_SOURCE_ROOT/pyproject.toml $CI_TEST_ROOT
+}
+
+install_test_dependencies () {
+    pip install -e ".[ert]"
+    pip install pytest
+}
+
+start_tests () {
```

Comment:
This can also be skipped since it is equivalent to 
https://github.com/equinor/komodo/blob/b6be86ded198956312e8304a4729ef628355997b/ci/func_def.sh#L33-L35